base/scopedvalues.jl

```diff
@@ -56,7 +56,7 @@ Base.isassigned(val::ScopedValue) = val.has_default
 
 const ScopeStorage = Base.PersistentDict{ScopedValue, Any}
 
-mutable struct Scope
+struct Scope
```

Comment:
Actually I think the reason why I made this `mutable` was to avoid eager copies in L120.
Let me check my benchmarks,

Reply:
The benchmark seem neutral.